fpga/pulpissimo-zcu106/elf_run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+trap "exit" INT TERM
+trap "kill 0" EXIT
+
+
+SCRIPTDIR=$(dirname $0)
+UART_TTY=${PULP_ZCU104_UART_TTY:=/dev/ttyUSB0}
+UART_BAUDRATE=${PULP_ZCU104_UART_BAUDRATE:=115200}
+
+#Execute gdb and connect to openocd via pipe
+$OPENOCD/bin/openocd -f $SCRIPTDIR/openocd-zcu104.cfg &
```

Comment:
Please make sure to replace all zcu104 remain with the zcu106 equivalents. I guess a global string replace in the pulpissimo-zcu106 folder should do the trick